apps/DemoApp/DemoAppUITests/DemoAppUITests.m

```diff
@@ -43,6 +43,17 @@ - (void)testPlayVideo {
     }
 }
 
+- (void)testPlayLivestreamVideo {
+    XCUIApplication *app = [[XCUIApplication alloc] init];
+    [app setLaunchEnvironment:@{@"ENV_KEY": envKey, @"TEST_SCENARIO": @"LIVESTREAM"}];
```

Comment:
thanks for adding this! Perhaps let's call it `LL-HLS` to distinguish from "regular live"?

Reply:
i'll add another test that is LL